go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cyberark/secretless-broker
 require (
 	github.com/aws/aws-sdk-go v1.15.79
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/denisenkom/go-mssqldb v0.0.0-20191001013358-cfbb681360f0
+	github.com/denisenkom/go-mssqldb v0.12.3
```

Comment:
This is fine, but not needed (there's a replace further down that shifts it to the sub-repo of our fork - that's why Snyk can't scan this file.)